lib/shell.js

```diff
@@ -34,6 +34,8 @@ module.exports = function(cmd, args, options, callback) {
         }
       };
 
+  process.setMaxListeners(0);
```

Comment:
It's a band-aid for a different symptom (listeners not being cleared, etc.).  Fine for now, but we should try to figure out why it's triggering those warnings in the first place.  Elsewhere, I found that using anonymous functions (unnamed) were causing this, as they couldn't be removed from the event listeners.